apps/base-docs/tutorials/docs/4_smart-wallet-and-rainbowkit.md

```diff
@@ -0,0 +1,207 @@
+
+![Default Connection](../../assets/images/smart-wallet/rainbowkit-default.png)
+
+What about the smart wallet? Isn't it supposed to work automatically?
+
+It does, but only if the user **does not** have the browser extension installed. Open a private window with extensions disabled and try again. Now, you will be directed to use the smart wallet to log in.
+
+![Default Connection](../../assets/images/smart-wallet/rainbow-smart-wallet.png)
+
+As mentioned above, this experience isn't bad for users of one type of wallet or the other, but it makes things difficult for users who are using both types of wallet and may want to choose one or the other when interacting with your app. It also might not be the listing or ordering of wallets you prefer.
+
+## Customizing the List of Wallets
+
+To fix this UI/UX problem, you can create a [custom wallet list]. Open `src/wagmi.ts`:
+
+- Change the list imported networks to `base` and `baseSepolia`
+- Import `connectorsForWallets` from RainbowKit, instead of `getDefaultConfig`
+- Import the `coinbaseWallet`, and any other wallets you wish to support
+- Import `http` and `createConfig` from wagmi
+
+```tsx
+import { http, createConfig } from 'wagmi';
+import { base, baseSepolia } from 'wagmi/chains';
+import {
+  coinbaseWallet,
+  metaMaskWallet,
+  rainbowWallet,
+  walletConnectWallet,
+} from '@rainbow-me/rainbowkit/wallets';
+import { connectorsForWallets } from '@rainbow-me/rainbowkit';
+```
+
+Next, use `connectorsForWallets` to create a list of wallets, organized in the groups you prefer. You'll need to get a `projectId` from [WalletConnect] if you don't have one already.
+
+```tsx
+const connectors = connectorsForWallets(
+  [
+    {
+      groupName: 'Recommended',
+      wallets: [coinbaseWallet],
+    },
+    {
+      groupName: 'Popular',
+      wallets: [rainbowWallet, metaMaskWallet],
+    },
+    {
+      groupName: 'Wallet Connect',
+      wallets: [walletConnectWallet],
+    },
+  ],
+  {
+    appName: 'Your App Name',
+    projectId: '<YOUR WALLETCONNECT PROJECT ID>',
+  },
+);
+```
+
+Finally, export your `config` using the `connectors` and networks you've selected.
+
+```tsx
+export const config = createConfig({
+  connectors,
+  chains: [base, baseSepolia],
+  ssr: true,
+  transports: {
+    [base.id]: http(),
+    [baseSepolia.id]: http(),
+  },
+});
+```
+
+Now, the connection experience contains the wallets in the order you selected!
+
+![Customized wallet list](../../assets/images/smart-wallet/customized-rainbow.png)
+
+---
+
+## Tuning the Coinbase Wallet Connection
+
+:::danger
+
+If you've already connected with the site, the below won't work unless you clear site data! It will appear that the flag does nothing.
```

Comment:
Maybe nudge them on how to clear site data?

Reply:
https://support.google.com/accounts/answer/32050?hl=en&co=GENIE.Platform%3DDesktop